sources/web/datalab/polymer/test/file-browser-test.ts

```diff
@@ -194,5 +196,59 @@ window.addEventListener('WebComponentsReady', () => {
       singleSelectionEnabledButtonIds.forEach((id) => assertEnabledState(id, false));
       multiSelectionEnabledButtonIds.forEach((id) => assertEnabledState(id, true));
     });
+
+    it('creates new notebook correctly', async () => {
```

Comment:
This is perhaps too picky, but these three tests do not quite test what their descriptions say they do. You are testing to see if the file-browser correctly calls the underlying mechanism that performs these actions, but you are not actually testing to see that those actions were correctly done. But unless you can come up with a concise wording, then it's acceptable as is.

Reply:
No I agree. Let me try to make it more correct.